ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
+  disabled: boolean;
+  required: boolean;
+  autocomplete: string | null;
+  allowDropdown: boolean;
+  autoPlaceholder: 'aggressive' | 'off' | 'polite' | undefined;
```

Comment:
I see that `'aggressive' | 'off' | 'polite'` is used in at least two diferent places. Would it be worth having a custom type for it?

Reply:
We have a lint rule that prevents us from using aliases which means we actually shouldn't create custom types just to make code cleaner